include/solvers/mpm_base.tcc

```diff
@@ -51,6 +51,21 @@ mpm::MPMBase<Tdim>::MPMBase(const std::shared_ptr<IO>& io) : mpm::MPM(io) {
     if (analysis_.find("locate_particles") != analysis_.end())
       locate_particles_ = analysis_["locate_particles"].template get<bool>();
 
+    // Stress rate method (None/Jaumann)
+    try {
+      if (analysis_.find("stress_rate") != analysis_.end()) {
+        if (analysis_["stress_rate"].template get<std::string>() == "jaumann")
+          stress_rate_ = mpm::StressRate::Jaumann;
```

Comment:
To test these lines, can you make two of the functions in `write_mesh_particles_unitcell.cc` have a "stress_rate" entry? Perhaps one "jaumann" and one "error" to trigger the warning?

Reply:
added!